wasmbinding/test/custom_query_test.go

```diff
@@ -116,8 +116,8 @@ func (suite *CustomQuerierTestSuite) TestInterchainQueryResultNotFound() {
 	}
 	resp := icqtypes.QueryRegisteredQueryResultResponse{}
 	err := suite.queryCustom(ctx, contractAddress, query, &resp)
-	expectedErrMag := "Generic error: Querier contract error: codespace: interchainqueries, code: 1115: query wasm contract failed"
-	suite.Require().Errorf(err, expectedErrMag)
+	expectedErrMsg := "Generic error: Querier contract error: codespace: interchainqueries, code: 1115: query wasm contract failed"
+	suite.Require().ErrorContains(err, expectedErrMsg)
```

Comment:
`Errorf` also works. I am curious about your reasoning to use `ErrorContains` instead.

Reply:
Errorf just tests any error, but i want to test a particular one